AirSkyBoat/AirSkyBoat
# Fix several various bugs

**_I affirm:_**


## Please enter a player-facing description
- The status effect bind from the spell bind now has a retail-accurate random duration between 1 and 60 seconds (if it is not fully resisted). The full duration is drawn from a normal distribution with mean of 30 seconds and std. dev. of 12 seconds and then can be half resisted to half the duration. (Tracent)
- Earthen Ward should now correctly overwrite a stoneskin effect previously applied by Earthen Ward. (Tracent)
- The poison from the fly ability Venom now has retail accurate power of 1 HP / tick and  retail accurate max duration of 240 seconds. (Tracent)

## What does this pull request do? (Please be technical)
- The dynamis currency is simply an incorrect item number in the drop SQL, need to be fixed both here and in Horizon specific SQL. Thus the player-facing description will be in that PR.
- Fixes some dyna mob name spelling errors
- The bind fix is based on retail testing by Tracent (Faust showed normal distribution with mean=21.6 std. dev.=11.6 , Huge Hornet showed normal distribution with mean=24.6 std. dev.=10.3). Thus very little if any impact of level, macc, etc.
- The Earthen Ward fix requires special code in the Earthen Ward Lua file because of the different overwrite behaviors of the stoneskin from Earthen Ward and the stoneskin from the spell stoneskin.

## Steps to test these changes
- Fight orcs in Dynamis-Qufim and they should now drop the correct currency.
- Repeatedly cast bind and see that duration should behave as normal distribution with mean of 21 seconds and std. dev. of 12 seconds
- Try all combinations of applying stoneskin via the spell stoneskin (A) and stoneskin via Earthen Ward (B). A should not overwrite A, A should overwrite B, B should not overwrite A, B should overwrite B.
- Flight dragonflies for example and see that poison from venom is only 1 HP / tick and last max of 240 seconds.

## Special Deployment Considerations
